### PR TITLE
Add basic EspoCRM support

### PR DESCRIPTION
**Pull request by Hofundur**
Adds a minimal integration to EspoCRM which add minimal Toggl Button in bradcrumbs on each detailed view.

### CODE REVIEW

**Comment by bryanph** [comment]
@Hofundur can you please add some pictures of where you positioned the buttons?

**Comment by Hofundur** [comment]
przechwytywanie
przechwytywanie1

File: src/scripts/origins.js [hunk 1]
```diff
@@ -138,6 +138,10 @@ export default {
     url: '*://*.esa.io/*',
     name: 'Esa.io'
   },
+  EspoCRM: {
+    url: '',
+    name: 'espocrm'
```
Comment (tcrammond) [inline src/scripts/origins.js]: Please format the `name` as `EspoCRM`

File: src/scripts/origins.js [hunk 2]
```diff
@@ -138,6 +138,10 @@ export default {
     url: '*://*.esa.io/*',
     name: 'Esa.io'
   },
+  EspoCRM: {
+    url: '',
```
Comment (tcrammond) [inline src/scripts/origins.js]: As there is also a cloud version of EspoCRM available, we can define a url of `*://*.espocloud.eu/*'`

**Comment by tcrammond** [comment]
Hey, we're cleaning up our backlog and since this PR has been inactive for a while, we're going to close it for now. Anyone please feel free to revisit it some time, and we will gladly review it again 🙇

**Pull request closed without merging**